Day-1/Notebook/JavaScript Notebook Extensions.ipynb

```diff
@@ -34,16 +34,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Tempering with Notebook app"
+    "## Tampering with Notebook app"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The first tool that is availlable to you and that you shoudl be aware of are browser \"developpers tool\". The exact naming can change across browser, and might require the installation of extensions. But basically they can allow you to inspect/modify the DOM, and interact with the javascript code that run the frontend.\n",
+    "The first tool that is available to you and that you should be aware of is the browser \"developers tool\". The exact naming can change across browser, and might require the installation of extensions. But basically they can allow you to inspect/modify the DOM, and interact with the javascript code that run the frontend.\n",
     "\n",
-    " - In Chrome and safari Developper tools are in the menu [Put mmenu name in english here] \n",
+    " - In Chrome and safari Developer tools are in the menu (More tools, Developer tools)\n",
     " - In firefox you might need to install [Firebug](http://getfirebug.com/)\n",
```

Comment:
Firefox has good developer tools available directly from the main menu now. I know this is unrelated, but may as well fix it now we're looking at this file.